eclass/cmake-utils.eclass

```diff
@@ -682,14 +729,11 @@ _cmake_ninja_src_make() {
 
 	[[ -e build.ninja ]] || die "build.ninja not found. Error during configure stage."
 
-	_ninjaopts_from_makeopts
+	[ -z "${NINJAOPTS}" ] && NINJAOPTS="$(_ninjaopts_from_makeopts)"
```

Comment:
Please use a bash conditional `[[ ]]` rather than the `test` built-in command `[ ]`.

Also you should check to see if NINJAOPTS is set, rather than checking to see that it is non-empty. If I have `NINJAOPTS=""` in make.conf, that should be respected. For example:

```
[[ ${NINJAOPTS+x} == x ]] || NINJAOPTS="$(_ninjaopts_from_makeopts)"
```